integreat_cms/cms/models/permission_manager/permission_manager.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class PermissionManager(models.Model):
```

Comment:
I guess I am fine with introducing some dummy model for additional permissions, but I would prefer the name `CustomPermissionManager`, otherwise it's kinda suggesting that it takes care of everything.

Could you also change the directory name into `custom_permissions` to match the naming?

Reply:
Done 👍